docs/src/content/docs/usage-guide/index.mdx

```diff
@@ -50,9 +53,51 @@ as a Pre-Commit hook. So, the following steps will help you with the process:
    git commit -m 'chore: added a dummy .gitkeep file'
    ```
 
-If Crisp was setup properly, you will see either a passing/failing message on
-STDOUT. If the output is a failing message then your commit message obviously
-does not follow the
+### Approach 2 (RECOMMENDED)
+
+1. Ensure Pre-Commit is locally installed and accessible (following the
+   [official installation guidelines](https://pre-commit.com#installation)).
+
+2. Create a `.pre-commit-config.yaml` file with the following content:
+
+   ```yaml
+   default_install_hook_types:
+    - pre-commit
+    - commit-msg
+
+    repos:
+      - repo: https://github.com/pre-commit/pre-commit-hooks
+        rev: v5.0.0
+        hooks:
+          - id: trailing-whitespace
+          - id: end-of-file-fixer
+          - id: check-yaml
+          - id: check-added-large-files
+
+    - repo: https://github.com/Weburz/crisp
+      rev: "v1.0.0"
+      hooks:
+        - id: crisp
+          name: lint commit messages
+   ```
+
+3. Setup the hooks for your project by executing:
+
+   ```console
+   pre-commit install --install-hooks
+   ```
+
+4. To test out whether Crisp is working as part of your Pre-Commit hooks, try
+   adding a dummy commit like so:
+
+   ```console
+   touch .gitkeep
+   git add .gitkeep
+   git commit -m 'chore: added a dummy .gitkeep file'
+   ```
+If Crisp was setup properly using any of the above approaches, you will see either
+a passing/failing message on STDOUT. If the output is a failing message then your
+commit message obviously does not follow the
 [Conventional Commits specifications](https://conventionalcommits.org) and you
 will have to improve it accordingly.
```

Comment:
The recommended approach should **ALWAYS** be the first thing the user should be reading followed by the other available options. So, maybe move this section of the docs up and organise the documentation accordingly?